src/pages/Profile/Queue/Queue.jsx

```diff
@@ -25,8 +24,9 @@ export default function Queue({ queues, setQueues, setSettings }) {
 
     async function handleDeleteMember(id) {
         try {
-            const wrappedQueueId = queues.find(item => { return (item.wrap == true) })._id
-            const membersWithDelete = members.filter(item => { return (item.id != id) })
+            const wrappedQueueId = queues.find(item => { return (item.wrap == true) })._id;
+            queues.find(item => { return (item.wrap == true) }).units.find(item => { return (item.id == id) }).active = false;
+            const membersWithDelete = queues.find(item => { return (item.wrap == true) }).units;
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
            const membersWithDelete = queues.find((item) => { return (item.wrap == true) }).units;
```